dashboard/templates/base.html

```diff
@@ -0,0 +1,166 @@
+<!DOCTYPE html>
+<!-- git commit: {{ g.build_info.git_commit }} -->
+<!-- git branch: {{ g.build_info.git_branch }} -->
+<!-- GitHub Action name: {{ g.build_info.github_action_name }} -->
+<!-- GitHub Action run url: {{ g.build_info.github_action_run_url }} -->
+<html lang="en" class="govuk-template">
+<head>
+    <meta charset="utf-8">
+    <title>
+        {% if form and form.errors %}Error: {% endif %}
+        {% block title %}{% endblock %}
+        - View and Download Gender Recognition Certificates
```

Comment:
This could be `GRC stats` to match the nav bar title

Reply:
Changed the title